apps/accounts/forms/update.py

```diff
@@ -129,3 +133,29 @@ def save(self, user, *args, **kwargs):
             }
             rollbar.report_exc_info(request=request, extra_data=extra_data)
         return user
+
+
+class TimeZoneForm(AbstractModelInstanceUpdateForm):
+    class Meta:
+        model = UserProfile
+        fields = (
+            'timezone',
+        )
+
+    def __init__(self, instance, *args, **kwargs):
+        """Initialization for TimeZoneForm.
+        """
+        user = instance
+        user_profile = user.profile
+        super(TimeZoneForm, self).__init__(user, *args, **kwargs)
+
+        self.fields['timezone'].initial = user_profile.timezone
+        self.fields['timezone'].label = ''
+
+    def save(self, request, *args, **kwargs):
+        user = super(TimeZoneForm, self).save(request, *args, **kwargs)
+        user_profile = user.profile
+        timezone = request.POST.get('timezone', '')
+        user_profile.timezone = timezone if timezone else 'America/Los_Angeles'
```

Comment:
The fallback here should be something like:

```
from htk.utils import htk_setting
...
default_timezone = htk_setting('HTK_DEFAULT_TIMEZONE')
user_profile.timezone = timezone or default_timezone
```

Reply:
Done